frontend-react/src/AppRouter.tsx

```diff
@@ -248,6 +248,14 @@ export const appRoutes: RouteObject[] = [
                     },
                 ],
             },
+            {
+                path: "/product/release-notes",
```

Comment:
Should this path be content/about/release-notes?

Reply:
Following the pattern of current route: https://reportstream.cdc.gov/product/release-notes